tests/OSPSuite.HelpersForTests/ModuleHelperForSpecs.cs

```diff
@@ -60,11 +96,15 @@ private SpatialStructure getSpatialStructureModule1()
          var bw = newConstantParameter(BW, 20);
          organism.Add(bw);
 
-
-         var organismMoleculeProperties = _objectBaseFactory.Create<IContainer>().WithName(MOLECULE_PROPERTIES)
+         var globalMoleculeProperties = _objectBaseFactory.Create<IContainer>().WithName(MOLECULE_PROPERTIES)
             .WithMode(ContainerMode.Logical)
             .WithContainerType(ContainerType.Molecule);
-         organism.Add(organismMoleculeProperties);
+
+         //add two parameters. One will be replaced when merging the second module
+         globalMoleculeProperties.Add(newConstantParameter("P1", 10));
```

Comment:
In this use case, P1  and P2 are set in module 1 and P1 and P3 are set in module 2 with a different value for P1. Test is checking that the last value wins